Explorer/Assets/DCL/Landscape/TerrainGenerator.cs

```diff
@@ -194,6 +198,21 @@ public void SwitchVisibility(bool isVisible)
             }
         }
 
+        private async Task ReEnableTerrainAsync(AsyncLoadProcessReport processReport)
+        {
```

Comment:
I would still keep it in Utils and re-use for Worlds, so they don't deviate too much and we will be able to merge them in future. But not critical for now

Reply:
ReEnableTerrainAsync